app/lib/frontend/templates/views/pkg/install_tab.dart

```diff
@@ -0,0 +1,150 @@
+  required List<String>? tags,
+  required bool isDevDependency,
+}) {
+  final packageName = version.package;
+  List<_ImportExample> importExamples;
+  if (version.libraries!.contains('$packageName.dart')) {
+    importExamples = <_ImportExample>[
+      _ImportExample(packageName, '$packageName.dart'),
+    ];
+  } else {
+    importExamples = version.libraries!
+        .map((library) => _ImportExample(packageName, library))
+        .toList();
+  }
+
+  final executables = version.pubspec!.executables?.keys.toList();
+  executables?.sort();
+  final hasExecutables = executables != null && executables.isNotEmpty;
+  final useAsLibrary = !hasExecutables || importExamples.isNotEmpty;
+
+  return d.fragment([
+    if (hasExecutables) _useAsExecutable(version.package, executables!),
+    if (useAsLibrary) _useAsLibrary(version, tags, isDevDependency),
+    if (importExamples.isNotEmpty) _useAsImport(importExamples),
+  ]);
+}
+
+d.Node _useAsExecutable(String package, List<String> executables) {
+  return d.fragment([
+    d.h2(text: 'Use this package as an executable'),
+    d.h3(text: 'Install it'),
+    d.p(text: 'You can install the package from the command line:'),
+    d.pre(
+      attributes: {'data-text-to-copy': 'dart pub global activate $package'},
+      child: d.code(
+        classes: ['language-shell'],
+        child: d.strong(text: 'dart pub global activate $package'),
+      ),
+    ),
+    d.h3(text: 'Use it'),
+    d.p(text: 'The package has the following executables:'),
+    d.pre(
+      child: d.code(
+        classes: ['language-shell'],
+        children: executables.map(
+          (e) => d.fragment([
+            d.text(r'$ '),
+            d.strong(text: e),
+            d.text('\n'),
+          ]),
+        ),
+      ),
+    ),
+  ]);
+}
+
+d.Node _useAsLibrary(
+  PackageVersion version,
+  List<String>? tags,
+  bool isDevDependency,
+) {
+  final isFlutterPackage = version.pubspec!.usesFlutter;
+  final bool usePubGet = !isFlutterPackage ||
+      tags == null ||
+      tags.isEmpty ||
+      tags.contains(SdkTag.sdkDart);
+  final bool useFlutterPackagesGet =
+      isFlutterPackage || (tags != null && tags.contains(SdkTag.sdkFlutter));
+  final pubAddCommand =
+      'pub add ${version.package}${isDevDependency ? ' --dev' : ''}';
+  final exampleDepKey = isDevDependency ? 'dev_dependencies' : 'dependencies';
+  final showEditorSupport = usePubGet || useFlutterPackagesGet;
+  final flutterOnly = useFlutterPackagesGet && !usePubGet;
+
+  return d.fragment([
+    d.h2(text: 'Use this package as a library'),
+    d.h3(text: 'Depend on it'),
+    d.p(text: 'Run this command:'),
+    if (usePubGet) _pubAddNode('Dart', 'dart $pubAddCommand'),
+    if (useFlutterPackagesGet) _pubAddNode('Flutter', 'flutter $pubAddCommand'),
+    d.p(children: [
+      d.text('This will add a line like this to your package\'s '
+          'pubspec.yaml (and run an implicit '),
+      d.code(
+          classes: ['language-shell'],
```

Comment:
Does this have any effect, when it's not inside a pre?

Reply:
Looks like hljs doesn't extend it with its own attributes, so I'll just remove it.